build/common.sh

```diff
@@ -100,20 +100,20 @@ kube::build::get_docker_wrapped_binaries() {
         );;
     "arm")
         local targets=(
-          cloud-controller-manager,armel/busybox
-          kube-apiserver,armel/busybox
-          kube-controller-manager,armel/busybox
-          kube-scheduler,armel/busybox
-          kube-aggregator,armel/busybox
+          cloud-controller-manager,arm32v7/busybox
```

Comment:
We should generally only use GOARMv7 things. alpine is a special case as they only have armv6 or armv8. Hence we have to drop down to armv6 for alpine (only)